packages/p2p-v2/src/components/AdvertsTableRow/AdvertsTableRow.tsx

```diff
@@ -0,0 +1,192 @@
+
+const BASE_CURRENCY = 'USD';
+
+const AdvertsTableRow = memo((props: TAdvertsTableRowRenderer) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const { data: exchangeRateValue, subscribe } = useExchangeRateSubscription();
+    const { isDesktop, isMobile } = useDevice();
+    const history = useHistory();
+    const isBuySellPage = getCurrentRoute() === 'buy-sell';
+
+    const { data: paymentMethods } = p2p.paymentMethods.useGet();
+    const { data } = p2p.advertiser.useGetInfo() || {};
+    const { daily_buy = 0, daily_buy_limit = 0, daily_sell = 0, daily_sell_limit = 0 } = data || {};
+
+    const {
+        account_currency,
+        advertiser_details,
+        counterparty_type,
+        effective_rate,
+        local_currency,
+        max_order_amount_limit_display,
+        min_order_amount_limit_display,
+        payment_method_names,
+        price_display,
+        rate,
+        rate_type,
+    } = props;
+
+    useEffect(() => {
+        if (local_currency) {
+            subscribe({
+                base_currency: BASE_CURRENCY,
+                target_currency: local_currency,
+            });
+        }
+    }, [local_currency, subscribe]);
+
+    const exchangeRate = exchangeRateValue?.rates?.[local_currency ?? ''];
+    const Container = isMobile ? 'div' : Fragment;
+
+    const { completed_orders_count, id, is_online, name, rating_average, rating_count } = advertiser_details || {};
+
+    const { displayEffectiveRate, effectiveRate } = generateEffectiveRate({
+        exchangeRate,
+        localCurrency: local_currency,
+        marketRate: Number(effective_rate),
+        price: Number(price_display),
+        rate,
+        rateType: rate_type,
+    });
+    const hasRating = !!rating_average && !!rating_count;
+    const isBuyAdvert = counterparty_type === BUY_SELL.BUY;
+    const isMyAdvert = data?.id === id;
+    const ratingAverageDecimal = rating_average ? Number(rating_average).toFixed(1) : null;
+    const textColor = isMobile ? 'less-prominent' : 'general';
+
+    return (
+        <div
+            className={clsx('p2p-v2-adverts-table-row', {
+                'p2p-v2-adverts-table-row--advertiser': !isBuySellPage,
+            })}
+        >
+            <Container>
+                {isBuySellPage && (
+                    <div
+                        className='flex flex-row gap-4 items-center cursor-pointer'
+                        onClick={() => history.push(`/cashier/p2p-v2/advertiser?id=${id}`)}
+                    >
+                        <UserAvatar
+                            isOnline={is_online}
+                            nickname={name || ''}
+                            showOnlineStatus
+                            size={25}
+                            textSize='xs'
+                        />
+                        <div className='flex flex-col'>
+                            <div
+                                className={clsx('flex flex-row items-center gap-2', {
```

Comment:
just a doubt.. do we need to specify the flex-row here? isn't the direction row by default?

Reply:
let me check on this 👍